src/Umbraco.Web.UI.Client/src/less/components/application/umb-app-a11y.less

```diff
@@ -0,0 +1,10 @@
+﻿.sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0,0,0,0);
+    border: 0;
+}
```

Comment:
@RachBreeze it seems this class is more or less identical with the class `visually-hidden` in this PR https://github.com/umbraco/Umbraco-CMS/pull/5544/files

The A11Y Project mention the `visually-hidden`.
https://a11yproject.com/posts/how-to-hide-content/

Not sure which is is preferable?
/cc @MMasey

Reply:
Hi @bjarnef sr-only is a naming convention in bootstrap (https://getbootstrap.com/docs/4.0/utilities/screenreaders/).  

There are a three pull requests now for a screen reader only class this one and #5585 #5544 

It might be worth just doing a PR for screen readonly class and then updating these PRs to reference that class.  What do you think @MMasey  and @nul800sebastiaan?

Reply:
Have a PR pending which will just be CSS amends, the class  can either be `visually-hidden` or `sr-only`

Indeed almost did the PR with  `visually-hidden` .  However bootstrap also has ` sr-only-focusable` class  which 

> will ensure that the link becomes visible once focused (for sighted keyboard users).

Happy to call the class either, the list of issues on #5277 does require a class that supports hidden text for screen readers.  It would be good to standardise on one name, and get a PR accepted with just that class in.  Let me know which  you prefer and I will submit a PR for that